libs/editor/WordPressEditor/src/main/java/org/wordpress/android/editor/EditorMediaUtils.java

```diff
@@ -44,10 +44,10 @@ public static BitmapDrawable getAztecPlaceholderDrawableFromResID(Context contex
     }
 
     public static int getMaximumThumbnailSizeForEditor(Context context) {
-        Point size = DisplayUtils.getDisplayPixelSize(context);
-        int screenWidth = size.x;
-        int screenHeight = size.y;
-        int maximumThumbnailWidthForEditor = screenWidth > screenHeight ? screenHeight : screenWidth;
+        Rect size = DisplayUtils.getWindowSize(context);
+        int screenWidth = size.height();
+        int screenHeight = size.width();
```

Comment:
Can we recheck the assignments: `size.height()` to `screenWidth` and `size.width()` to `screenHeight`?

Reply:
@ashiagr Oops, I missed that, Done in [03f1d52](https://github.com/wordpress-mobile/WordPress-Android/pull/16677/commits/03f1d52899bf75d168b4e911e28c81e62e4a3221)